hbase-server/src/main/java/org/apache/hadoop/hbase/regionserver/storefiletracker/StoreFileListFile.java

```diff
@@ -174,7 +176,13 @@ private NavigableMap<Long, List<Path>> listFiles() throws IOException {
         LOG.warn("Found invalid track file {}, which is not a file", file);
         continue;
       }
-      if (!TRACK_FILE_PATTERN.matcher(file.getName()).matches()) {
+      if (TRACK_FILE_OLD_PATTERN.matcher(file.getName()).matches()) {
+        Path newFile = new Path(trackFileDir,
+          file.getName() + TRACK_FILE_SEPARATOR + System.currentTimeMillis());
+        LOG.info("Found invalid track file {}, Rename to {}", file, newFile);
+        fs.rename(file, newFile);
```

Comment:
As mentioned in the above comment, here if the rename fails, we need to to deal with it...

Reply:
I modified the test to check if StoreFileListFile without a timestamp can be read correctly.